extensions/amp-analytics/0.1/vendors.js

```diff
@@ -41,25 +41,30 @@ export const ANALYTICS_CONFIG = {
       'scrollHeight': 'SCROLL_HEIGHT',
       'screenWidth': 'SCREEN_WIDTH',
       'screenHeight': 'SCREEN_HEIGHT'
+      // Vars that GA can use: documentEncoding, userLanguage
     }
     // TODO(btownsend, #871): Add a generic hit format to make custom analytics
     // easier.
   },
 
   'googleanalytics': {
+    'vars': {
+      'eventValue': "0",
+      'documentLocation': 'AMPDOC_URL'
```

Comment:
If I do that, people will have to override the ampdocUrl property to set a different value for dl. That looks and feel counter-intuitive. SInce this is a common thing many people do, I opted for this solution instead.